kaltura_admin.py

```diff
@@ -53,21 +55,26 @@ def get_kmc_admin_list(kaltura_user_id, kmcs):
         filter.loginEnabledEqual = KalturaNullableBoolean.TRUE_VALUE
         pager = KalturaFilterPager()
         pager.pageSize = 500
-        result = client.user.list(filter, pager)
-        log.info(
-            f"Retriving kmc: {kmc_name} Total Records: {result.totalCount}")
-        for item in result.objects:
-            if item.roleNames == "Publisher Administrator" or item.roleNames == "Content Manager":
-                kmc_user_id = item.id.lower()
-                if '@' in kmc_user_id:
-                    # remove @xxxxx
-                    item_id = kmc_user_id.partition('@')
-                    id = item_id[0]
-                    kmc_user_id = id
-
-                # add to user_id_list
-                user_id_list.append(kmc_user_id)
-
+        try:
+            result = client.user.list(filter, pager)
+            log.info(
+                f"Retriving kmc: {kmc_name} Total Records: {result.totalCount}")
+            for item in result.objects:
+                if item.roleNames == "Publisher Administrator" or item.roleNames == "Content Manager":
+                    kmc_user_id = item.id.lower()
+                    if '@' in kmc_user_id:
+                        # remove @xxxxx
+                        item_id = kmc_user_id.partition('@')
+                        id = item_id[0]
+                        kmc_user_id = id
+
+                    # add to user_id_list
+                    user_id_list.append(kmc_user_id)
+        except KalturaException as ke:
```

Comment:
Probably don't need the more specific `KalturaException` unless you were doing something different here. The message looks the same.

Reply:
Done. Remove the block for KalturaException